addons/account/models/account_tax.py

```diff
@@ -224,6 +239,34 @@ def _compute_tax_group_id(self):
                 ('country_id', '=', False),
             ], limit=1)
 
+    def _compute_is_used(self):
+        taxes_in_transactions_ctr = (
+            Counter(dict(self.env['account.move.line']._read_group([], groupby=['tax_ids'], aggregates=['__count']))) +
+            Counter(dict(self.env['account.reconcile.model.line']._read_group([], groupby=['tax_ids'], aggregates=['__count'])))
+        )
+
+        installed_models = self.env['ir.model'].sudo().search_fetch([('model', 'in', list(OTHER_MODELS_USING_TAXES.keys()))], ['model'], limit=len(OTHER_MODELS_USING_TAXES))
```

Comment:
hum... this isn't the odoo way

We'd rather define a function, inherited in the different modules, each of them adding their own model to the list of models we have to check. 
The benefit is that each module then contains all the code and logic it is adding. And it's arguably cleaner :)

Reply:
Indeed, probably forgotten leftovers from testings!